.github/workflows/ci.yml

```diff
@@ -0,0 +1,16 @@
+on: push
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby: [2.2, 2.3, 2.4, 2.5, 2.6, 2.7, jruby, jruby-head, truffleruby, truffleruby-head]
```

Comment:
I decided to drop travis as only rubinius was tested there an not on GHA, and it was already marked as allowed to fail, so not really supported.